### PR TITLE
Updates due to pkg changes

### PR DESCRIPTION
**Pull request by nachocano**
- Change in pkg to allow setting up the domain when reporting custom metrics otherwise they assume `knative.dev` domain.

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -21,7 +21,7 @@ required = [
 
 [[override]]
   name = "knative.dev/pkg"
-  branch = "release-0.9"
+  branch = "master"
```
Comment (n3wscott) [inline Gopkg.toml]: /hold until the next release is cut

**Comment by nachocano** [comment]
@n3wscott for lgtm/approval

**Pull request merged**